DllImportGenerator/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -229,13 +234,15 @@ private static bool IsGeneratedDllImportAttribute(AttributeSyntax attrSyntaxMayb
                 || attrName.EndsWith(PrefixedGeneratedDllImportAttribute);
         }
 
-        private AttributeSyntax ProcessGeneratedDllImportAttribute(
+        private void ProcessGeneratedDllImportAttribute(
             IMethodSymbol method,
             AttributeData attrData,
             bool generateForwarders,
-            out DllImportStub.GeneratedDllImportData dllImportData)
+            out DllImportStub.GeneratedDllImportData stubDllImportData,
+            out DllImportStub.GeneratedDllImportData targetDllImportData)
```

Comment:
Would we be able have this just return `stubDllImportData` and convert that to `targetDllImportData` outside of this function?